### DISCUSSION TITLE
How to handle multiple feature files in single webdriverIO project

### DISCUSSION THREAD

**Discussion by Digitalteamautomation** [webdriverio/webdriverio#10281] [Q&A]
I tried adding feature file and made necessary configuration in config file.
When i try to run the project, i got reference error.
So i reverted my changes and tried to run . Code which was running earlier is not working and it becomes like corrupted.
It automatically skips the steps in feature file.

<img width="686" alt="image" src="https://user-images.githubusercontent.com/131886499/234896149-9e574f72-06ee-4122-9c3e-c0bc233ab39e.png">
<img width="694" alt="image" src="https://user-images.githubusercontent.com/131886499/234896441-b39a273f-0253-4493-8d69-7558c30eddb7.png">

**Comment by christian-bromann** [comment]
Do you have a minimal reproducible example?

**Reply by Digitalteamautomation** [reply]
these are confidential . let me create a mock and share the example. Meanwhile,
Can you show me the correct steps in handling multiple feature files in webdriverIO with cucumber framework..

**Reply by christian-bromann** [reply]
What do you mean by that?
